packages/ui-focusable/src/Focusable/index.tsx

```diff
@@ -194,34 +194,31 @@ class Focusable extends Component<FocusableProps> {
   }
 
   get focusable() {
-    let focusable = findFocusable(this, () => true, true) || []
-    const focusableCount = (focusable && focusable.length) || 0
+    const focusableArr = findFocusable(this, () => true, true) || []
+    const focusableCount = (focusableArr && focusableArr.length) || 0
 
     warn(
       focusableCount === 1,
       `[Focusable] Exactly one focusable child is required (${focusableCount} found).`
     )
 
-    // @ts-expect-error ts-migrate(2322) FIXME: Type 'unknown' is not assignable to type 'unknown[... Remove this comment to see the full error message
-    focusable = focusable ? focusable[0] : false
```

Comment:
Again some mess with types. I've introduced a new variable for the array of `focusable`s